### ISSUE TITLE
IstioOperator auto-generated docs out of order, values definition missing

### ISSUE THREAD

**Issue by ostromart** [istio/istio.io#7824]
The definitions are again alphabetically sorted which makes them very hard to read - this seems to be a regression since we had turned off sorting at some point. 
Also the [values definitions](https://github.com/istio/istio/blob/master/operator/pkg/apis/istio/v1alpha1/values_types.proto) seem to have disappeared. 
Tentatively assigning to Eric since he worked in that area recently.

**Comment by ericvn** [comment]
@ostromart For the values definitions, it looks like that page would gen to https://github.com/istio/istio/blob/master/operator/pkg/apis/istio/v1alpha1/v1alpha1.pb.html and looking at that file, there is no `location:` specified in the doc, so there would be no files in the doc repo.

**Comment by ericvn** [comment]
Also, not sure what you mean by definitions are again sorted alphabetically. Can you point me to what you mean?

I know there is an issue where types? where not being sorted so they came out randomly (like json, ...). This caused the docs to be repeatedly updated since their order was non-deterministic. I don't think this is what you mean, but wanted to make sure.

**Comment by ostromart** [comment]
Yes I guess it's random, which is even worse: https://istio.io/latest/docs/reference/config/istio.operator.v1alpha1/
At one point we had this in the same order as the proto which is what's desired. Does anyone own the docs infra these days? Sorry if I pulled you into this as an innocent bystander.

**Comment by ericvn** [comment]
@frankbu @davidhauck are probably the best bets for infra items, but I'm not sure how the .proto files get converted to the pb.html. So that page comes from https://github.com/istio/api/blob/release-1.7/operator/v1alpha1/istio.operator.v1alpha1.pb.html which is gen'd from https://github.com/istio/api/blob/release-1.7/operator/v1alpha1/operator.proto.
